t/bad_subclass.t

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env perl
+
+use lib 'lib', 't/lib';
+
+use Test2::V0;
+
+# This test verifies the error handling when a TCM class extends a
+# non-TCM subclass.  All this happens at compile time, so the usual
+# exception testing is not applicable.  The modules get a directory of
+# their own because they (intentionally) break TCM::Load.
```

Comment:
I'm not sure what TCM::Load has to do with anything. It's not being used in this test code.